rogue-thi-app/components/cards/TimetableCard.js

```diff
@@ -32,7 +34,7 @@ export default function TimetableCard () {
       }
     }
     load()
-  }, [router])
+  }, [router, time])
```

Comment:
Do we really need to refresh the timetable data just to refresh how the (probably identical) data is displayed in the UI?
I assume the API caching makes sure we dont really refresh it and dont spam the API.

Reply:
Yes you are right. I updated the code. The rendering of the remaining time is now updated every 30 seconds without calling the API again.
I think the time span is a good compromise of near-current data and website energy consumption.